package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ion-study-template",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w",
+    "lint": "eslint . && web-ext lint",
+    "test": "npm run test-addon"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^16.0.0",
+    "@rollup/plugin-node-resolve": "^10.0.0",
+    "eslint": "^7.12.1",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
I _think_ you can remove this devDep since it's commented out everywhere above.

Reply:
Yeah, this PR is still WIP and not ready for review, as I'm currently working on it!